core/src/main/java/org/mskcc/cbio/portal/scripts/ImportClinicalData.java

```diff
@@ -496,6 +496,8 @@ public static void main(String[] args) throws Exception {
 	               "meta (description) file" ).withOptionalArg().describedAs( "meta_file.txt" ).ofType( String.class );
 	        OptionSpec<String> study = parser.accepts("study",
 	                "cancer study id").withOptionalArg().describedAs("study").ofType(String.class);
+	        OptionSpec<String> attributeFlag = parser.accepts("a",
+	                "Flag for using MIXED_ATTRIBUTES").withOptionalArg().describedAs("a").ofType(String.class);
```

Comment:
I would further describe this as a "deprecated" option, not to encourage others to start using it since we actually want to go for the split samples / patients format.